src/append/rolling_file/rolling.rs

```diff
@@ -90,16 +87,17 @@ impl RollingFileWriterBuilder {
     #[must_use]
     pub const fn new() -> Self {
         Self {
-            rotation: Rotation::Never,
+            rotation: TimeRotation::Never,
             prefix: None,
             suffix: None,
             max_size: usize::MAX,
             max_files: None,
+            clock: None,
```

Comment:
Just default to `DefaultClock`?

Reply:
Updated, and I simplified the Clock related code as well.